<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Каталог Книг</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PB_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_csv">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Выгрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_add">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_del">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Автору</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Произведению</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Книге</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Месту</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Изданию</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Году</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Страницам</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISBN</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать по:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>802</width>
      <height>350</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Произведение</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Автор</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Книга</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Стеллаж/Полка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Издание</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Страницы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ISBN</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="PB_update">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>50</y>
      <width>209</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Добро пожаловать в Каталог книг!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_save">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
